english/red-pill/red-pill.md

```diff
@@ -0,0 +1,3 @@
+An wrinkly old man with a funny hat appears from nowhere.  He approaches you, removes his hat and removes a red pill from within it which he hands to you with a glass of chocolate milk. Reluctantly, you take the pill and place it on your tounge then drink the milk.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.